src/main/java/core/basesyntax/service/DefaultFileService.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class DefaultFileService implements FileService {
+    @Override
+    public String readFile(String filePath) {
+        try {
+            return Files.readString(Path.of(filePath));
```

Comment:
Better use `Files.readAllLines` and return List. It will reduce your work in ParseService